src/zeit/content/cp/automatic.py

```diff
@@ -1,4 +1,7 @@
 from zeit.content.cp.interfaces import IAutomaticTeaserBlock
+from zeit.solr import query as lq
```

Comment:
Hehe. Das ist glaub ich historisch, `lq` für "lucene query". Da es als Sprach-Hilfsmittel benutzt wird, find ich kurz aber schon wichtig. Ich kuck mal, ob man es vielleicht lokal in der jeweiligen Funktion machen kann, á la `Q = zeit.solr.query`, analog zu `E = lxml.objectify.E`.